src/config.rs

```diff
@@ -410,16 +405,11 @@ impl Config {
         let _ = f.read_to_string(&mut toml)?;
 
         // Parse the configuration file.
-        let toml = if let Value::Table(toml) =
-            toml.parse::<Value>()
-                .unwrap_or_else(|e| {
-                    print_error(format!("There was an error parsing the config.toml file: {}", e));
-                    exit(Error::Parse.into());
-                }) {
+        let toml = if let Value::Table(toml) = toml.parse::<Value>()? {
```

Comment:
We should probably chain the error here, just in case. Something like:
```rust
toml.parse::<Value>().chain_err(|| "there was an error parsing the config.toml file")?
```
This gives much better backtracing.